modular_skyrat/modules/milker/code/milker.dm

```diff
@@ -0,0 +1,305 @@
+					blind_message = span_purple("You hear a plapping noise."),
+					vision_distance = 1)
+
+	if(!do_mob(milker, being_milked, 1 SECONDS))
+		milker.visible_message(
+		message = span_purple("[milker] was interrupted!"),
+		self_message = span_purple("You were interrupted!"),
+		blind_message = span_purple("You hear someone's lips slip."),
+		vision_distance = 1)
+		return FALSE
+	if(!breastCheck(being_milked, milker, breasts)) // Their breasts may have changed state (gutted, clothes, drained...)
+		return FALSE
+
+	milker.adjustArousal(arousal_amt)
+
+	switch(handle_mode)
+		if(HANDLE_LIGHT) // Gentle paws...
+			if(milker.combat_mode) // Nibble...
+				milker.visible_message(
+				message = span_purple("[milker] nibbles on [self_suckle ? "[milker.p_their()] own" : "[being_milked]'s"] nipple, teasing out a stream of milk!"),
+				self_message = span_purple("You nibble into [self_suckle ? "your own" : "[being_milked]'s"] nipple and feel a stream of milk spray into your mouth!"),
+				blind_message = span_purple("You hear a nibble, and a squirt."),
+				vision_distance = 1)
+				being_milked.adjustPain(pain_amt)
+				playsound(milker.loc,'sound/weapons/bite.ogg', rand(10,50), TRUE)
+			else
+				milker.visible_message(
+				message = span_purple("[milker] suckles out a gentle stream of [self_suckle ? "[milker.p_their()] own" : "[being_milked]'s"] milk!"),
+				self_message = span_purple("You suckle down a gentle stream of [self_suckle ? "your own" : "[being_milked]'s"] milk!"),
+				blind_message = span_purple("You hear a slurp."),
+				vision_distance = 1)
+				being_milked.adjustArousal(arousal_amt)
+				being_milked.adjustPleasure(pleasure_amt)
+				playsound(milker.loc,'sound/items/drink.ogg', rand(10,50), TRUE)
+		if(HANDLE_STRONG) // SQUISH
+			if(milker.combat_mode) // CHOMP
+				milker.visible_message(
+				message = span_purple("[milker] bites down into [self_suckle ? "[milker.p_their()] own" : "[being_milked]'s"] nipple, slurping down a mouthful of milk!"),
+				self_message = span_purple("You chomp down into [self_suckle ? "your own" : "[being_milked]'s"] breast and feast upon the milk within!"),
+				blind_message = span_purple("You hear a hungry nibble."),
+				vision_distance = 1)
+				being_milked.adjustPain(pain_amt)
+				playsound(milker.loc,'sound/weapons/bite.ogg', rand(10,50), TRUE)
+			else
+				milker.visible_message(
+				message = span_purple("[milker] suckles down a mouthful of [self_suckle ? "[milker.p_their()] own" : "[being_milked]'s"] milk!"),
+				self_message = span_purple("You suckle down a mouthful of [self_suckle ? "your own" : "[being_milked]'s"] milk!"),
+				blind_message = span_purple("You hear a faint slurp."),
+				vision_distance = 1)
+				being_milked.adjustArousal(arousal_amt)
+				being_milked.adjustPleasure(pleasure_amt)
+				playsound(milker.loc,'sound/items/drink.ogg', rand(10,50), TRUE)
+
+	log_combat(milker, being_milked, "fed_boob", breasts.internal_fluids.log_list())
```

Comment:
I feel like we should create a new log file for this kind of stuff because I really don't want to see that when reading someone's combat logs.

Reply:
~~I do though~~ I dont really know how to do that, so I just removed the logging for these.